metals/src/main/scala/scala/meta/internal/metals/MultilineStringFormattingProvider.scala

```diff
@@ -159,14 +159,24 @@ final class MultilineStringFormattingProvider(
           case Some('|') =>
             val secondPipeIndex = lineText.indexOf('|', firstPipeIndex + 1)
             val secondPipePos = new Position(line, secondPipeIndex)
-            new TextEdit(new Range(zeroPos, secondPipePos), defaultIndent)
+            val textEdit =
+              new TextEdit(new Range(zeroPos, secondPipePos), defaultIndent)
+            Some(textEdit)
           case _ =>
             val pipePos = new Position(line, firstPipeIndex)
-            new TextEdit(new Range(zeroPos, pipePos), defaultIndent)
+            val textEdit =
+              new TextEdit(new Range(zeroPos, pipePos), defaultIndent)
+            Some(textEdit)
         }
       case _ =>
-        val newText = defaultIndent + "|"
-        new TextEdit(new Range(zeroPos, zeroPos), newText)
+        val isFirstLineOfMultiLine = lineText.trim.contains("\"\"\"|")
```

Comment:
What happens if the line doesn’t have a pipe? It’s optional

Reply:
If there is no pipe on the line it gets filtered out beforehand and the line never enters the `formatPipeLine` method. I suppose I should have added a test for this as well. I did just check it locally and that indeed is how it works.

Reply:
I just added another test for this https://github.com/scalameta/metals/pull/1249